### PR TITLE
#94 Issue/adapt colors sizes

### PR DESCRIPTION
**Pull request by stanislav-slavov**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -131,7 +131,7 @@
 <style>
 
   html {
-    font-size: 135%;
+    font-size: 125%;
```
Comment (thepiwo) [inline src/App.vue]: why not 100%?

Comment (stanislav-slavov) [reply src/App.vue]: 1rem on 100% is equal to 16px. 125% is 20px - It is alot easier to calculate.

File: src/components/layout/LeftSection.vue [hunk 2]
```diff
@@ -41,32 +44,32 @@
             {{stats.total_tips_length}}
           </div>
           <div class="overview__label">
-            Tips
+            Tips Aggregated
           </div>
         </div>
-        <div class="overview__item" v-if="stats.comments">
+        <!-- <div class="overview__item" v-if="stats.comments">
```
Comment (thepiwo) [inline src/components/layout/LeftSection.vue]: why is the stats that were required by yani removed?

File: src/views/TipsList.vue [hunk 3]
```diff
@@ -171,9 +165,14 @@
...
   }
   .container.wrapper{
+    padding-top: 0;
     min-height: 4rem;
   }
    .actions__container{
+      top: 0;
+      position: sticky;
+      position: -webkit-sticky;
+      z-index: 10;
```
Comment (thepiwo) [inline src/views/TipsList.vue]: why z-index?

File: src/views/TipsList.vue [hunk 4]
```diff
@@ -171,9 +165,14 @@
     }
   }
   .container.wrapper{
+    padding-top: 0;
     min-height: 4rem;
   }
    .actions__container{
+      top: 0;
+      position: sticky;
+      position: -webkit-sticky;
```
Comment (thepiwo) [inline src/views/TipsList.vue]: if not supported widely or with polyfil we shouldn't use it

File: src/views/UserProfileView.vue [hunk 5]
```diff
@@ -268,14 +265,20 @@
...
-    .address{
-      color: $standard_font_color;
-    }
-    .count{
-      color: $light_font_color;
-      font-size: .6rem;
+    top: 0;
+    position: sticky;
+    position: -webkit-sticky;
+    z-index: 10;
```
Comment (thepiwo) [inline src/views/UserProfileView.vue]: why z-index?

File: src/components/layout/LeftSection.vue [hunk 6]
```diff
@@ -118,38 +121,46 @@
...
 
-      .navigation__item, .side__button {
-        font-size: .75rem;
+      .navigation__item {
+        font-size: 1rem;
         text-transform: capitalize;
-        margin-bottom: 0.7rem;
-
+        margin-bottom: 1rem;
+        a a{
```
Comment (thepiwo) [inline src/components/layout/LeftSection.vue]: nested a doesn't seem to make sense

File: src/views/UserProfileView.vue (hunk 5 above)
Comment (stanislav-slavov) [reply src/views/UserProfileView.vue]: Because Position sticky is fixed when scrolled and it goes under the content.

**Pull request merged**